numpy/lib/npyio.py

```diff
@@ -714,9 +714,10 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
         ``converters = {3: lambda s: float(s.strip() or 0)}``.  Default: None.
     skiprows : int, optional
         Skip the first `skiprows` lines; default: 0.
-    usecols : sequence, optional
+    usecols : int or sequence, optional
         Which columns to read, with 0 being the first.  For example,
-        ``usecols = (1,4,5)`` will extract the 2nd, 5th and 6th columns.
+        ``usecols = (1,4,5)`` will extract the 2nd, 5th and 6th columns and
+        ``usecols = 3`` or  ``usecols = (3,)`` will extract the third column.
```

Comment:
Hmmm, maybe you can rewrite this a bit? I think the typical thing would be to leave the first part as it was and only add the new stuff after the tag (not sure exactly how it looks like out of my head):
```
Old documentation

.. versionadded:: 1.11.0

Describe that you can also use a single integer.
```